Automattic/wp-calypso
# Themes: Use dynamic theme filters data

No visual changes.

Use data fetched from /theme-filters API endpoint for the Theme Showcase filter search. Utilises new selectors added in #13928 and #14060, and removes the static data and library `theme-filters.js`


**To Test**
* Check that the theme filters in the search box at /themes work as before
* Check that urls containing filters and verticals work as before, including logged-out
For example:
http://calypso.localhost:3000/themes/business
http://calypso.localhost:3000/themes/filter/one-column+classic-menu+clean+artwork
